### PR TITLE
MTDSA-14298: TYS changes for Delete an Employment Financial Details Submission

### PR DESCRIPTION
**Pull request by GazalNebHMRC**

### CODE REVIEW

File: app/v1/connectors/DeleteEmploymentFinancialDetailsConnector.scala [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+      correlationId: String): Future[DownstreamOutcome[Unit]] = {
+
+    import request._
+
+    val downstreamUri = if (taxYear.useTaxYearSpecificApi) {
+      TaxYearSpecificIfsUri[Unit](
+        s"income-tax/${taxYear.asTysDownstream}/income/employments/${nino.value}/$employmentId"
+      )
+    } else {
+      Api1661Uri[Unit](
```
Comment (tom-lloyd-hmrc) [inline app/v1/connectors/DeleteEmploymentFinancialDetailsConnector.scala:44]: In the controller the URI was a `Release6Uri`, but here it's `Api1661Uri`

File: app/v1/connectors/DeleteEmploymentFinancialDetailsConnector.scala [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+  def deleteEmploymentFinancialDetails(request: DeleteEmploymentFinancialDetailsRequest)(implicit
+      hc: HeaderCarrier,
+      ec: ExecutionContext,
+      correlationId: String): Future[DownstreamOutcome[Unit]] = {
+
+    import request._
+
+    val downstreamUri = if (taxYear.useTaxYearSpecificApi) {
+      TaxYearSpecificIfsUri[Unit](
+        s"income-tax/${taxYear.asTysDownstream}/income/employments/${nino.value}/$employmentId"
```
Comment (tom-lloyd-hmrc) [inline app/v1/connectors/DeleteEmploymentFinancialDetailsConnector.scala:41]: We can just use `$nino` as `Nino.toString` returns `value`

File: app/v1/connectors/DeleteEmploymentFinancialDetailsConnector.scala (hunk 1 above)
Comment (GazalNebHMRC) [reply app/v1/connectors/DeleteEmploymentFinancialDetailsConnector.scala]: I have updated now. Thank you!

File: app/v1/connectors/DeleteEmploymentFinancialDetailsConnector.scala (hunk 2 above)
Comment (GazalNebHMRC) [reply app/v1/connectors/DeleteEmploymentFinancialDetailsConnector.scala]: Updated.

**Review by tom-lloyd-hmrc — APPROVED** [review]

**Pull request merged**